recipes-core/packagegroups/packagegroup-core-boot-openwrt.bb

```diff
@@ -32,6 +32,7 @@ RDEPENDS_${PN} = "\
     base-passwd \
     busybox \
     bridge-utils \
+    tcpdump \
```

Comment:
You can probably leave it for now, but at some point a good distro / image debug package set would be nice. tcpdump, strace, perf, etc. Network related items. I'd have to look at other examples to see exactly how to do it though. Khem probably knows right away of a decent example.